pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.flipkart.poseidon</groupId>
     <artifactId>poseidon</artifactId>
-    <version>5.14.1</version>
+    <version>5.14.2-SNAPSHOT</version>
```

Comment:
Can you make it a release version, and update all files with this version?

Reply:
Sure, will change it to 5.14.2 but before that you don't need to test anything with snapshot version? Also what all files need to be changed?